### PR TITLE
move com.ibm.ws.session.store.common to its own bundle

### PR DESCRIPTION
**Pull request by njr-11**
Move the decoupled com.ibm.ws.session.store.common package to its own bundle, such that it will be usable by both the sessionDatabase feature as well as the sessionCache feature.

### CODE REVIEW

File: dev/com.ibm.ws.session.store/build.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<!--
```
Comment (aguibert) [inline dev/com.ibm.ws.session.store/build.xml]: this file can be removed

File: dev/com.ibm.ws.session.store/.gitignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+/bin/
```
Comment (aguibert) [inline dev/com.ibm.ws.session.store/.gitignore]: this file can be removed

File: dev/com.ibm.ws.session.store/build.xml (hunk 1 above)
Comment (njr-11) [reply dev/com.ibm.ws.session.store/build.xml]: Thanks for catching these.  I removed both.

**Comment by njr-11** [comment]
#build

**Comment by njr-11** [comment]
#build

**Comment by njr-11** [comment]
Closing this out because #1824 includes this same commit along with another

**Pull request closed without merging**